### DISCUSSION TITLE
[KubeTrigger] Roadmap已经实现到哪个版本了

### DISCUSSION THREAD

**Discussion by haima95** [kubevela/kubevela#5896] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/2991141/233590717-04b3f585-6604-4d91-be19-2e5447b0ba6e.png)
这些是已经实现了吗

**Comment by charlie0129** [comment, marked as answer]
`cron` has been implemented. `webhook` is in-progress. More documentation is still needed before we can mark that task as completed. Some underlying optimizations are being carried out currently.

**Reply by haima95** [reply]
很感谢你的回复。谢谢～
还有个疑问想请教，kubevela上添加kube-trigger插件后该如何使用吗？github的介绍看着好像都是采用kubectl命令，我本地用kubectl按照example的例子也都是失败，是不是我本地环境问题（本地用minikube，基于kuberneters的方式安装的kubevela）。很感谢～

**Reply by charlie0129** [reply]
Can you refer to [KubeVela 中文社区会议（2023-2-21）]( https://b23.tv/yMJowS1) for some more up-to-date instructions and see if it works? There have been some changes going on, so the readme might need some updates.

**Reply by haima95** [reply]
Thx

**Reply by FogDong** [reply]
Hi， @haima95 ，你可以重新提个 issue，带上 ”kube-trigger“ 无法使用的例子，然后我们看下。
